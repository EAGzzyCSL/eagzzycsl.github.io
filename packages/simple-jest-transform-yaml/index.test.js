// FIXME: test.js 没有适配 lint 规则
/* global test, expect */
const yamlDemo = require('./yaml-demo.yaml')

test('transform-yaml', () => {
  expect(yamlDemo).toMatchSnapshot()
})
