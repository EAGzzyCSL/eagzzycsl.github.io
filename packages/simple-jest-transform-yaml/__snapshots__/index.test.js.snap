// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`transform-yaml 1`] = `
Object {
  "fruits": Array [
    "apple",
    "banana",
    "cherry",
  ],
}
`;
